pkg/lib/console.css

```
@@ -57,6 +57,6 @@
 }
 
 /* Ensure the console fits to its container (and doesn't attempt to go beyond the limits) */
```

If we want to enhance the comment:

```suggestion
/* Ensure the console fits to its container (and doesn't attempt to go beyond the limits).
   This is needed as the widget hardcodes a pixel value for width that can be too wide. */
```